docs/admin/kubelet-tls-bootstrapping.md

```diff
@@ -17,11 +17,18 @@ and progress on the feature is being tracked as [feature #43](https://github.com
 
 ## kube-apiserver configuration
 
-You must provide a token file which specifies at least one "bootstrap token" assigned to a kubelet bootstrap-specific group.
+You must provide a token which specifies at least one "bootstrap token" assigned to a kubelet bootstrap-specific group.
```

Comment:
This is awkward.  How about:
```
You must configure a token based authentication mechanism that will provide authorization for a user in the `system:bootstrappers` group.
```

Note that this group is hard coded as part of the bootstrap token mechanism but is configurable with the (should be deprecated) token auth file.  We should standardize the rest of the instructions in this file to that group.

Specifically, change references to `system:kubelet-bootstrap` or `kubelet-bootstrap-token` to `system:bootstrappers` everywhere below.  (Right now, even, the instructions are wrong as the CSV example uses one group but the RBAC rules use another.)